src/amo/components/AddonReview/styles.scss

```diff
@@ -10,6 +10,12 @@
 
   .Rating {
     margin: 6px 0 12px;
+    padding: 0 10px;
```

Comment:
The values in this file are inconsistent but I was just preventing regressions. This component is going away soon.